contribs/github-bot/internal/config/config.go

```diff
@@ -56,6 +56,17 @@ func Config(gh *client.GitHub) ([]AutomaticCheck, []ManualCheck) {
 				),
 			),
 		},
+		{
+			Description: "Changes related to gnoweb must be reviewed by its codeowners",
+			If: c.And(
+				c.BaseBranch("^master$"),
+				c.FileChanged(gh, "^gno.land/pkg/gnoweb/"),
+			),
+			Then: r.Or(
+				r.ReviewByUser(gh, "alexiscolin"),
+				r.ReviewByUser(gh, "gfanton"),
```

Comment:
This should have actually been either a review or the author being one of the two...

Reply:
I hesitated when creating this rule to do something similar as the `docs` one, I know that both want to be assigned as reviewers and review each other. So I think that even if one of them makes a PR (e.g., gfanton), it should be reviewed by the other (e.g., alexiscolin). 
No problem making the change in any case.